### ISSUE TITLE
SVG Icon for vscode-icons

### ISSUE THREAD

**Issue by anthony0030** [presidentbeef/brakeman#1746]
Could you please provide an SVG icon for vscode-icons:
https://github.com/vscode-icons/vscode-icons/issues/3141

**Comment by presidentbeef** [comment]
Yep: https://github.com/presidentbeef/brakeman/issues/1639#issuecomment-956408167

**Comment by anthony0030** [comment]
Thanks
